### ISSUE TITLE
Confusing error message when forgetting the secret

### ISSUE THREAD

**Issue by tr7zw** [Kir-Antipov/mc-publish#40]
Ran into the error ``Warning: 🗿 You didn't specify any targets, your assets have not been published`` with the exact same setup that worked on another project. In the middle of writing a big Discussions Q/A post, I remembered that I forgot to set up the secrets for this repo. The build should just fail in case you have a ``modrinth-id``/``curseforge-id`` defined, but the corresponding token is empty/invalid(since it clearly was forgotten).

**Comment by Kir-Antipov** [comment]
Didn't think anyone will ever get this warning, and this is definitely not the case when it should be shown. Yup, will rework some things for such scenarios

**Comment by Kir-Antipov** [comment]
> The build should just fail in case you have a `modrinth-id`/`curseforge-id` defined, but the corresponding token is empty/invalid(since it clearly was forgotten)

I ultimately chose not to implement this. The reason being, it's unclear when IDs are specified and when they are not. For instance, all my personal configs look like this:

```yaml
- name: Upload assets to GitHub, Modrinth, and CurseForge
  uses: Kir-Antipov/mc-publish@v3.2
  with:
    name: ""
    modrinth-token: ${{ secrets.MODRINTH_TOKEN }}
    curseforge-token: ${{ secrets.CURSEFORGE_TOKEN }}
    github-token: ${{ secrets.GITHUB_TOKEN }}
```

Does this mean I don't want to publish my mod to Modrinth, CurseForge, or GitHub because I didn't specify IDs for Modrinth and CurseForge and a tag for GitHub? No, of course not. Those can be found in my `fabric.mod.json`. 

Conversely, does it mean that someone who didn't specify a token for, let's say, Modrinth, but has the ID of their Modrinth project somewhere in their mod metadata for unspecified reasons, wants to publish the mod to Modrinth? Well, no, that's not always true either.

Consequently, this isn't a reliable indicator of the user's intentions. The only accurate one is a specified or unspecified token. As a result, I've simply revised the warning message that caused confusion. It now reads: 

> Warning: No valid platform tokens found in your config. To publish your project, please add the required access tokens for the desired platforms. Assets will not be published without them. Refer to the documentation for assistance in setting up your tokens.

I hope this provides more clarity.

**Comment by Kir-Antipov** [comment]
Implemented in [v3.3](https://github.com/Kir-Antipov/mc-publish/releases/tag/v3.3).
